app/helpers/pre_built_pages_helper.rb

```diff
@@ -1,15 +1,13 @@
 module PreBuiltPagesHelper
-  def include_page_content(path, default = nil)
-    page = Page.find_by(path: path)
-    return default if page.blank?
-    page.render_content
-  end
+  def include_or_create_page_content(path, message, options = {})
+    options.reverse_merge!({
+      edit_link: true
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.